content/components/icon.mdx

```diff
@@ -0,0 +1,214 @@
+---
+title: Icon
+react: https://primer.style/octicons/packages/react
+rails: https://primer.style/octicons/packages/ruby
+figma: https://www.figma.com/file/1ljgTFkT5NKNRfq5hw07JQ/Octicons?node-id=1%3A2&t=Lz5afoq1bbWrD9Yt-1
+description: Our icons are located in a separate library called Octicons, which is available in various implementations including React, Figma, Rails, and Styled System.
```

Comment:
It feels a little strange to start off by saying where the Icons are located (especially if we're not linking to it here). It looks like you've got it covered in the text below. 

```suggestion
description: Icons at GitHub are called Octicons, which are available in various implementations including React, Figma, Rails, and Styled System. 
```